### ISSUE TITLE
2.9.5. Brainstorming

### ISSUE THREAD

**Issue by Pauletta** [Jugendhackt/Handbuch-Jugend-Hackathons#12]
- Pfeile vor Zwischenüberschriften einfügen (s. PDF)
- Zwischenüberschriften farbig

**Comment by rhotep** [comment]
Da müssen wir uns nochmal zusammensetzten. Ich kann am Code nicht erkennen welche Überschriften als Zwischenüberschriften zählen und welche davon Pfeile haben sollen :)

**Comment by Pauletta** [comment]
Ja, ich verstehe, was du meinst. Das Brainstorming kapitel ist nochmal stärker unterteilt. Ich finde  es super schön, wie Martina das im Papier handbuch gelöst hat und dachte, das klappt vielleicht auch online? Also so, wie hier: http://handbuch.jugendhackt.de/appendix/00%20Handbuch_Jugend-Hackathons.pdf (Kapitel 3.9.5, ab Seite 33). Wir können uns aber trotzdem gerne mal im X-Hain verabreden ;)
